pkg/deploy/server/server_deployment.go

```diff
@@ -404,115 +403,58 @@ func GetFullCheServerImageLink(checluster *orgv1.CheCluster) string {
 	return imageParts[0] + ":" + checluster.Spec.Server.CheImageTag
 }
 
-func MountBitBucketOAuthConfig(deployContext *deploy.DeployContext, deployment *appsv1.Deployment) error {
-	secrets, err := deploy.GetSecrets(deployContext, map[string]string{
-		deploy.KubernetesPartOfLabelKey:    deploy.CheEclipseOrg,
-		deploy.KubernetesComponentLabelKey: deploy.OAuthScmConfiguration,
-	}, map[string]string{
-		deploy.CheEclipseOrgOAuthScmServer: "bitbucket",
-	})
-
-	if err != nil {
+func MountBitBucketOAuthConfig(ctx *deploy.DeployContext, deployment *appsv1.Deployment) error {
+	secret, err := getOAuthConfig(ctx, "bitbucket")
+	if secret == nil {
 		return err
-	} else if len(secrets) > 1 {
-		return errors.New("More than 1 BitBucket OAuth configuration secrets found")
-	} else if len(secrets) == 1 {
-		mountSecret(deployment, &secrets[0], deploy.BitBucketOAuthConfigMountPath)
-		mountEnv(deployment, []corev1.EnvVar{
-			{
-				Name:  "CHE_OAUTH1_BITBUCKET_CONSUMERKEYPATH",
-				Value: deploy.BitBucketOAuthConfigMountPath + "/" + deploy.BitBucketOAuthConfigConsumerKeyFileName,
-			}, {
-				Name:  "CHE_OAUTH1_BITBUCKET_PRIVATEKEYPATH",
-				Value: deploy.BitBucketOAuthConfigMountPath + "/" + deploy.BitBucketOAuthConfigPrivateKeyFileName,
-			},
-		})
-
-		endpoint := secrets[0].Annotations[deploy.CheEclipseOrgScmServerEndpoint]
-		if endpoint != "" {
-			mountEnv(deployment, []corev1.EnvVar{{
-				Name:  "CHE_OAUTH1_BITBUCKET_ENDPOINT",
-				Value: endpoint,
-			}})
-		}
 	}
 
+	mountVolumes(deployment, secret, deploy.BitBucketOAuthConfigMountPath)
+	mountEnv(deployment, "CHE_OAUTH1_BITBUCKET_CONSUMERKEYPATH", deploy.BitBucketOAuthConfigMountPath+"/"+deploy.BitBucketOAuthConfigConsumerKeyFileName)
+	mountEnv(deployment, "CHE_OAUTH1_BITBUCKET_PRIVATEKEYPATH", deploy.BitBucketOAuthConfigMountPath+"/"+deploy.BitBucketOAuthConfigPrivateKeyFileName)
+
+	oauthEndpoint := secret.Annotations[deploy.CheEclipseOrgScmServerEndpoint]
+	if oauthEndpoint != "" {
+		mountEnv(deployment, "CHE_OAUTH1_BITBUCKET_ENDPOINT", oauthEndpoint)
```

Comment:
Is it supposed to have only one `_` before `ENDPOINT`?

Reply:
Right, only one `_`